src/compute-types/src/plan/interpret/api.rs

```diff
@@ -374,13 +375,19 @@ where
                     key_val_plan,
                     plan,
                     input_key,
+                    mfp_after,
                 } => {
                     // Descend recursively into all children.
                     let input = self.apply_rec(input, rg)?;
                     // Interpret the current node.
-                    Ok(self
-                        .interpret
-                        .reduce(&self.ctx, input, key_val_plan, plan, input_key))
+                    Ok(self.interpret.reduce(
+                        &self.ctx,
+                        input,
+                        key_val_plan,
+                        plan,
+                        input_key,
+                        mfp_after,
+                    ))
```

Comment:
This seems uncovered and interesting

Reply:
We cannot cover these lines now, unfortunately, because LIR does not have a pure analysis in its refinements. The physical monotonicity refinement uses the `Interpreter` API, but ends up in `FoldMut` and not `Fold`. So we should be seeing these lines be uncovered in `main` also, I assume, since `Fold` is never used.